CHANGELOG.md

```diff
@@ -3,6 +3,40 @@
 Subscribe to the [tracking issue #2231](https://github.com/tigerbeetle/tigerbeetle/issues/2231)
 to receive notifications about breaking changes!
 
+## TigerBeetle 0.16.45
+
+Released: 2025-06-13
+
+### Features
+
+- [#2907](https://github.com/tigerbeetle/tigerbeetle/pull/2907)
+
+  Add documentation on how to monitor TigerBeetle, track requests end-to-end for better monitoring.
+
+- [#3019](https://github.com/tigerbeetle/tigerbeetle/pull/3019)
+
+  Improve the compatibility of our new CDC plugin by supporting signed integer types. Thanks 
+  @alvinyan-bond for your feedback!
+
+### Internals
+
+- [#3023](https://github.com/tigerbeetle/tigerbeetle/pull/3023)
+
+  Fix a liveness bug related to when replicas are syncing.
+
+- [#3022](https://github.com/tigerbeetle/tigerbeetle/pull/3022)
+
+  Fix a crash related to timing when measuring commit timing.
```

Comment:
I think these two belong in the safety section. We usually use internals for things like testing.

Reply:
Updated!

Reply:
Ah, mea culpa, I should have indicated in the commit message that this is a follow-up to https://github.com/tigerbeetle/tigerbeetle/pull/2907#event-18074142775!

This is a crash, but this code was never released, so this actually belongs to the _features_ section where we introduced the bug! That is, we should've filed  #2907 and #3022 under the same bullet point. 

But, yeah, it's much more obvious now, then when I was crafting commit message for #3022, sorry!